From: =?UTF-8?q?J=C3=A1n=20Tomko?= <jtomko@redhat.com>
Date: Mon, 11 Apr 2016 15:26:06 +0200
Subject: [PATCH] conf: reduce indentation in virDomainDefAddImplicitVideo

Return early if there is nothing to do.

(cherry picked from commit 1485be178461b6e9504d21216b7ef480f4980ffa)
---
 src/conf/domain_conf.c | 31 ++++++++++++++++---------------
 1 file changed, 16 insertions(+), 15 deletions(-)

diff --git a/src/conf/domain_conf.c b/src/conf/domain_conf.c
index cc99301..7c8fa4e 100644
--- a/src/conf/domain_conf.c
+++ b/src/conf/domain_conf.c
@@ -18572,22 +18572,23 @@ virDomainDefAddImplicitVideo(virDomainDefPtr def)
 
     /* For backwards compatibility, if no <video> tag is set but there
      * is a <graphics> tag, then we add a single video tag */
-    if (def->ngraphics && !def->nvideos) {
-        if (VIR_ALLOC(video) < 0)
-            goto cleanup;
-        video->type = virDomainVideoDefaultType(def);
-        if (video->type < 0) {
-            virReportError(VIR_ERR_INTERNAL_ERROR, "%s",
-                           _("cannot determine default video type"));
-            goto cleanup;
-        }
-        video->vram = virDomainVideoDefaultRAM(def, video->type);
-        video->heads = 1;
-        if (VIR_ALLOC_N(def->videos, 1) < 0)
-            goto cleanup;
-        def->videos[def->nvideos++] = video;
-        video = NULL;
+    if (def->ngraphics == 0 || def->nvideos > 0)
+        return 0;
+
+    if (VIR_ALLOC(video) < 0)
+        goto cleanup;
+    video->type = virDomainVideoDefaultType(def);
+    if (video->type < 0) {
+        virReportError(VIR_ERR_INTERNAL_ERROR, "%s",
+                       _("cannot determine default video type"));
+        goto cleanup;
     }
+    video->vram = virDomainVideoDefaultRAM(def, video->type);
+    video->heads = 1;
+    if (VIR_ALLOC_N(def->videos, 1) < 0)
+        goto cleanup;
+    def->videos[def->nvideos++] = video;
+    video = NULL;
 
     ret = 0;
  cleanup:
